usaspending_api/search/v2/views/spending_by_category.py

```diff
@@ -117,6 +120,8 @@ def post(self, request: Request) -> Response:
             response = FundingAgencyViewSet().perform_search(validated_payload, original_filters)
         elif validated_payload["category"] == "funding_subagency":
             response = FundingSubagencyViewSet().perform_search(validated_payload, original_filters)
+        elif validated_payload["category"] == "psc":
```

Comment:
In addition to this can you remove the `self.industry_and_other_codes()` logic in spending_by_category since PSC and NAICS are the last two.